src/js/actions/conversation-actions.js

```diff
@@ -0,0 +1,33 @@
+export const MESSAGE_ADDED = 'MESSAGE_ADDED';
+export const MESSAGES_CLEARED = 'MESSAGES_CLEARED';
+export const SET_CONVERSATION = 'SET_CONVERSATION';
+export const RESET_CONVERSATION = 'RESET_CONVERSATION';
+
+export function resetConversation() {
+  return {
+    type: RESET_CONVERSATION
+  }
+}
+
+export function setConversation(props) {
+  return {
+    type: SET_CONVERSATION,
+    conversation: props
+  }
+}
+
+export function messageAdded(props) {
+  return {
+    type: MESSAGE_ADDED,
+    message: Object.assign({
+      _id: Math.random(),
```

Comment:
Is it okay to use `Math.random()` for this? It'll only return numbers in the [0,1) range.

It's only used for the `key` prop of the `MessageComponent` for now, does React expect the `key` to be an integer?